### PR TITLE
Updated the style of error pages

### PR DESCRIPTION
**Pull request by hammerfunk**

### CODE REVIEW

**Comment by hammerfunk** [comment]
Yeah...?

Doing this task: Make custom error pages if we're offline with QAE Contact details

But just read again and saw priority was low...

**Comment by matthewford** [comment]
You're replacing key copy in the error message type - they are not all 400's

**Comment by hammerfunk** [comment]
I've left the content but added stuff around it

File: public/422.html [hunk 1]
```diff
@@ -2,14 +2,13 @@
 <html>
   <head>
     <meta http-equiv="content-type" content="text/html; charset=UTF-8">
-    <title>The change you wanted was rejected (422)</title>
```
Comment (matthewford) [inline public/422.html]: @hammerfunk check again. This line should not have been changed.

**Comment by hammerfunk** [comment]
Good spot! Thanks

**Pull request merged**